mirdata/dali.py

```diff
@@ -197,6 +210,8 @@ def download(data_home=None):
     if data_home is None:
         data_home = utils.get_default_dataset_path(DATASET_DIR)
 
+    download_utils.downloader(data_home, file_downloads=[METADATA_REMOTE])
+
     print(
```

Comment:
this module was written before we added the `info_message` argument to the `downloader` function. While we're here, could you move the print string to `info_message=...`?

Reply:
good idea. done